src/components/Hero/_Hero.scss

```diff
@@ -165,3 +165,16 @@
     }
   }
 }
+
+.hero--tertiary {
+  background-color: var(--ui-gray-xdark);
+
+  .hero__content {
+    @include wrapper;
+
+    color: var(--ui-white);
+    display: flex;
+    flex-flow: column nowrap;
+    padding: var(--space-l) var(--space);
+  }
```

Comment:
What happened to `.hero_heading`?  The spacing for that element still needs to be set to `space-stack-s` (16px or 1rem).

Reply:
`.hero_heading` was removed because running `npm run test`, it returns this error:
"180:5  ✖  Expected selector ".hero--tertiary .hero__content .hero__heading" to come before selector ".nypl--books-and-more  .hero--secondary .hero__content .hero__heading" - no-descending-specificity"
That wasn't an issue when `.hero--tertiary` was above `.hero--secondary`.  Would you happen to know how to resolve that?

Reply:
Hmm, I am not seeing any errors when I run `npm run test`.  Maybe kill the `node_modules` folder and run `npm install` before running the tests?  If the error persists, then just move the `tertiary` block back above the `secondary` block.